### PR TITLE
RDAPI-21234 Added information about Date interval behaviour for API Manager metrics

### PR DESCRIPTION
**Pull request by scormican**
…our for API Manager Metrics

Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added it to the correct Sprint milestone
* [ ] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

File: content/en/docs/apim_administration/apimgr_admin/api_mgmt_monitor.md [hunk 1]
```diff
@@ -84,6 +84,9 @@ In both the **API Usage** and **Application Usage** views, you can use the **F
 
 You can select a date interval instead of the default value of **Today** (for example, **Last 7 days**, **Last 30 days**, or a custom date range).
 
+* **Today** and **Specify date/time** - gives data up to the last timeframe processed i.e. includes today's data, up to last timeframe recorded.
+* All other Date intervals - full days but only up to the end of yesterday i.e. it does not include today's data.
```
Comment (andreamussap) [inline content/en/docs/apim_administration/apimgr_admin/api_mgmt_monitor.md:88]: @scormican what does **full days** mean?

Comment (jmbermingham) [reply content/en/docs/apim_administration/apimgr_admin/api_mgmt_monitor.md:88]: 'Full Day' means a 24 hour period, that is, the full day's worth of data (not partial which would be a less then 24 hour period)

**Review by andreamussap — APPROVED** [review]
https://deploy-preview-1134--axway-open-docs.netlify.app/docs/apim_administration/apimgr_admin/api_mgmt_monitor/#date-interval

**Comment by andreamussap** [comment]
@scormican Fair play for adding the Jira ticket to the title of the PR! I needed more information, and I was able to ask for help through the Jira ticket.

If you're working on GitHub (not Netlify CMS) remember to [Sync or delete an outdated fork before contributing](https://axway-open-docs.netlify.app/docs/contribution_guidelines/deleting_a_repository/)

**Pull request merged**